guides/client/stratum.mdx

```diff
@@ -130,46 +120,24 @@ Now that we've built the source, we can start our proxy. We recommend using a pr
 
 ```bash
 ./build/bin/go-quai-stratum --region=REGION-NAME --zone=ZONE-NAME
 ```
 
-Or to run with ports use:
-
-```sh
-./build/bin/go-quai-stratum --region=REGION-WS-PORT --zone=ZONE-WS-PORT
-```
-
 <Warning>
 	**Running the proxy will only work for chains your node is validating state for**. Global nodes validate state for all chains, whereas
 	slice nodes only validate state for the chains you specify.
 </Warning>
 
-Available options for `REGION-NAME`, `ZONE-NAME`, `REGION-PORT`, and `ZONE-PORT` can be found below. You must select a corresponding region and zone, i.e. paxos and paxos3 or hydra and hydra2.
-
-If you're running a version of `go-quai-stratum` prior to [v0.9.0-rc.0](https://github.com/dominant-strategies/go-quai-stratum/releases/tag/v0.9.0-rc.0), you'll need to pass in the corresponding web socket ports as chain names are not supported prior to this version.
-
-| Chain Name | Type   | Chain Index | Web Socket Port | Stratum Run Flags                |
-| ---------- | ------ | ----------- | --------------- | -------------------------------- |
-| Cyprus     | Region |             | 8002            | `--region=cyprus`                |
-| Paxos      | Region |             | 8003            | `--region=paxos`                 |
-| Hydra      | Region |             | 8004            | `--region=hydra`                 |
-| Cyprus1    | Zone   | [0 0]       | 8200            | `--region=cyprus --zone=cyprus1` |
-| Cyprus2    | Zone   | [0 1]       | 8201            | `--region=cyprus --zone=cyprus2` |
-| Cyprus3    | Zone   | [0 2]       | 8202            | `--region=cyprus --zone=cyprus3` |
-| Paxos1     | Zone   | [1 0]       | 8220            | `--region=paxos --zone=paxos1`   |
-| Paxos2     | Zone   | [1 1]       | 8221            | `--region=paxos --zone=paxos2`   |
-| Paxos3     | Zone   | [1 2]       | 8222            | `--region=paxos --zone=paxos3`   |
-| Hydra1     | Zone   | [2 0]       | 8240            | `--region=hydra --zone=hydra1`   |
-| Hydra2     | Zone   | [2 1]       | 8241            | `--region=hydra --zone=hydra2`   |
-| Hydra3     | Zone   | [2 2]       | 8242            | `--region=hydra --zone=hydra3`   |
+Available options for `REGION-NAME` and `ZONE-NAME` can be found below. You must select a corresponding region and zone, e.g. cyprus and cyprus1.
 
-<Warning>
-	**Do not open the above web socket ports except** in the specific case where your miner is on a different network than your node/stratum
-	(and even then, be sure to only open the port to the necessary machine). You may be putting your local network security at risk.
-</Warning>
+| Chain Name | Type   | Chain Index | Stratum Run Flags                |
+| ---------- | ------ | ----------- | -------------------------------- |
+| Cyprus     | Region |             | `--region=cyprus`                |
+| Cyprus1    | Zone   | [0 0]       | `--region=cyprus --zone=cyprus1` |
+| Cyprus2    | Zone   | [0 1]       | `--region=cyprus --zone=cyprus2` |
```

Comment:
The previous section seemed correct, any reason for this change?

Reply:
Because Golden Age will likely only be running 2 shards, all of the other shard commands were removed to create less confusion for users.

Reply:
Sure, FYI 2 is possible but not confirmed it could be 1.

However, I do think the port numbers were valuable, if you want to populate the config file you have to use the numbers directly, plus its useful in other ways.

Reply:
Ports added back to table in [0ea6b98](https://github.com/dominant-strategies/quai-docs/pull/59/commits/0ea6b983ba5aa36c0e2805bb0b929ee8ebc60936)